src/argus/notificationprofile/models.py

```diff
@@ -167,6 +167,7 @@ class Media(models.TextChoices):
     # TODO: support for multiple email addresses / phone numbers / Slack users
     media = MultiSelectField(choices=Media.choices, min_choices=1, default=Media.EMAIL)
     active = models.BooleanField(default=True)
+    phone_number = models.ForeignKey("argus_auth.PhoneNumber", on_delete=models.SET_NULL, blank=True, null=True)
```

Comment:
Should we change this `to` parameter to a reference, to be consistent with the other `ForeignKey` fields?
```suggestion
    phone_number = models.ForeignKey(to=PhoneNumber, on_delete=models.SET_NULL, blank=True, null=True)
```

Reply:
Actually, I'd rather get rid of "to", and use strings whenever we point to something in a different app. Lowers chance of import-loops, since the strings are IIRC lazily imported.